[en] update network/intro/issues (#4921)
DaoCloud/DaoCloud-docs: docs/en/docs/network/intro/issues.md

> ### Known issues in version 0.9

Reviewer: ```suggestion
### Known issues in v0.9
```
这种可全文统改

> #### An error occurred during the synchronization of spidercoordinator with the status, but its status remains as running

Reviewer: ```suggestion
#### An error occurred during spidercoordinator synchronization, but its status remains running
```

#### When kubeadm-config is missing, serviceCIDR cannot be obtained from kubeControllerManager Pod

In some scenarios, kubeadm is not used to create a cluster, so there may be no kubeadm-config configMap. It will try to obtain it from kubeControllerManager. However, due to a bug, serviceCIDR cannot be obtained from kubeControllerManager Pod, resulting in the failure of Spidercoordinator status update.

**Reference:**

https://github.com/spidernet-io/spiderpool/pull/3020

> #### Due to the addition of a new attribute  to the SpiderCoordinator CRD in v0.9.0, upgrading from v0.7.0 to this version will result in a panic.

Reviewer: ```suggestion
#### Upgrading from v0.7.0 to v0.9.0 will cause a panic due to a new attribute in the SpiderCoordinator CRD
```
Heading shall not be too long and have no period at end.

Spiderpool v0.9.0 adds a new attribute `TxQueueLen` to the SpiderCoordinator CRD, but it does not have a default value during the upgrade operation, which will cause panic. You need to use it and treat it as a default value of 0.

**Reference:**

https://github.com/spidernet-io/spiderpool/pull/3118

> #### Due to different cluster deployment methods, the SpiderCoordinator returns an empty serviceCIDR, which prevents the creation of Pods

Reviewer: ```suggestion
#### Different cluster deployment methods cause SpiderCoordinator to return an empty serviceCIDR, preventing Pod creation
```

Due to different cluster deployment methods, there are two types of CIDRs recorded in the cluster kube-controller-manager Pod: `Spec.Containers[0].Command` and `Spec.Containers[0].Args`. For example, the RKE2 cluster is `Spec.Containers[0].Args` instead of `Spec.Containers[0].Command`, and `Spec.Containers[0].Command` is hardcoded in the original logic, resulting in abnormal judgment, returning an empty serviceCIDR, and failing to create a Pod.

**References:**

https://github.com/spidernet-io/spiderpool/pull/3211

> ### Known Issues in version 0.8

Reviewer: ```suggestion
### Known Issues in v0.8
```

https://github.com/spidernet-io/spiderpool/pull/2538

> #### Third-party controllers, such as RedisCluster, cannot be recognized by Spiderpool, causing the Pods of StatefulSets controlled by them to be unable to have fixed IP addresses

Reviewer: ```suggestion
#### Spiderpool cannot recognize third-party controllers like RedisCluster, preventing StatefulSet Pods from having fixed IP addresses
```

https://github.com/spidernet-io/spiderpool/pull/2434

> #### In scenarios where there is a one-to-one mapping between Pods and IPs, IPAM allocation is blocked, leading to some Pods being unable to run so that affects the performance of IP allocation

Reviewer: ```suggestion
#### In one-to-one Pod-to-IP mapping scenarios, IPAM allocation is blocked, preventing some Pods from running and affecting IP allocation performance
```

https://github.com/spidernet-io/spiderpool/pull/2532

> #### IPPool.Spec.MultusName specifies the namespace/multusName, but due to an incorrect parsing of the namespace, affinity fails, so that the associated multusName cannot be found

Reviewer: ```suggestion
#### Incorrect namespace parsing in IPPool.Spec.MultusName causes affinity to fail, making the multusName unfindable
```
